Assets/Editor/E05_Controls.cs

```diff
@@ -77,6 +101,9 @@ void AddTask()
             {
                 m_TasksContainer.contentView.AddChild(CreateTask(m_TextField.text));
                 m_TextField.text = "";
+
+                // Give focus back to text field.
+                m_TasksContainer.focusController.SwitchFocus(m_TextField);
```

Comment:
In fact you are right we could also write m_TextField.Focus() here. It is not an extension method, it is part of the Focusable class.
However we need FocusController to be public to get access to the currently focused element.